third_party/penumbra/cmake/compiler-flags.cmake

```diff
@@ -57,7 +57,7 @@ ELSEIF ( CMAKE_COMPILER_IS_GNUCXX OR "${CMAKE_CXX_COMPILER_ID}" STREQUAL "AppleC
     ADD_CXX_DEFINITIONS("-pipe") # Faster compiler processing
     ADD_CXX_DEFINITIONS("-std=c++11") # Enable C++11 features in g++
     ADD_CXX_DEFINITIONS("-pedantic") # Turn on warnings about constructs/situations that may be non-portable or outside of the standard
-    ADD_CXX_DEFINITIONS("-ffor-scope")
+    #ADD_CXX_DEFINITIONS("-ffor-scope")
```

Comment:
@nealkruis Just removing this deprecated flag, shouldn't have any functional change unless you were doing something funny with variables defined inside `for` blocks.

Reply:
No problem. Pretty sure this was just borrowed from the EnergyPlus flag settings.

Reply:
Yes, I took it from EnergyPlus flag settings.